Jellyfin.Api/Controllers/ItemsController.cs

```diff
@@ -400,9 +401,9 @@ public class ItemsController : BaseJellyfinApiController
                 }
 
                 // Filter by Series Status
-                if (!string.IsNullOrEmpty(seriesStatus))
+                if (seriesStatus.Length != 0)
                 {
-                    query.SeriesStatuses = seriesStatus.Split(',').Select(d => (SeriesStatus)Enum.Parse(typeof(SeriesStatus), d, true)).ToArray();
+                    query.SeriesStatuses = seriesStatus;
```

Comment:
Looks like the original also did some validation on the series status,

Reply:
The middleware does a similar thing already, except it doesn't throw if an invalid enum value is provided.